### ISSUE TITLE
Accessibility check

### ISSUE THREAD

**Issue by Faithchidindu** [oppia/oppia-android#5362]
![oppia 2](https://user-images.githubusercontent.com/127211589/229183700-cca398f8-542a-405e-9e69-03dc75a11f34.jpg)
![oppia 1](https://user-images.githubusercontent.com/127211589/229185493-06cc16f5-ccc4-4c44-9374-5f27f9b501fe.jpg)

The “stories you can play” text was not quite visible. The learner might struggle to read it. The text “stories you can play’ was given a white colour on a grey background so it did not pass the WCAG accessibility check.
It would be better if the text was given a black colour. 
White text should be used on a dark background, dark text should be used on white background inorder to make the text easy to read

There is inconsistency in the UI theme. Some of the screens are in light mode while some are in dark mode. There is no uniformity. The user interface will look much better if the user is allowed to switch between light and dark themes. This creates a sense of uniformity, reliability and control

**Comment by ShubhadeepKarmakar** [comment]
I can't reproduce this issue/bug.
On my side:
## Light mode
![accessibility2](https://github.com/oppia/oppia-android/assets/99060332/d358d87b-aaf3-4193-8953-40ddf81a26ac)
## Dark mode
![accessibility](https://github.com/oppia/oppia-android/assets/99060332/787911a8-60f8-4e21-99ae-9a45bb0a2741)


I have checked the XML files, and everything is fine. Additionally, both ['topic_lessons_title.xml'](https://github.com/oppia/oppia-android/blob/d0c8b81566c956c6edf6a7c19c14f8c8391dd939/app/src/main/res/layout/topic_lessons_title.xml#L19) and ['topic_lessons_story_summary.xml'](https://github.com/oppia/oppia-android/blob/d0c8b81566c956c6edf6a7c19c14f8c8391dd939/app/src/main/res/layout/topic_lessons_story_summary.xml#L116) use the same component color, namely ['component_color_shared_primary_text_color'](https://github.com/oppia/oppia-android/blob/d0c8b81566c956c6edf6a7c19c14f8c8391dd939/app/src/main/res/values/component_colors.xml#L104). This means that if the text color for 'Stories You Can Play' is white, then the text color for 'Jaime's Adventures in the Arcade' would also be white.

@Faithchidindu I think some code changes have occurred on your side, which is why you are facing this issue.

**Comment by adhiamboperes** [comment]
Thanks @ShubhadeepKarmakar!

Given the time when this issue was filed, it is likely that this was encountered on an older version of the app, before we released full darkmode support. So I am closing this and adding in to 0.13 release blockers for verification.
